python_modules/dagster/dagster_tests/execution_tests/pipes_tests/test_merge_streams.py

```diff
@@ -47,11 +48,40 @@ def _process_log_stream(stream: Iterator[bytes]) -> Iterator[LogItem]:
 
 
 def _is_kube_timestamp(maybe_timestamp: str) -> bool:
```

Comment:
I don't feel so good about duplicating this pretty complex function here and in dagster-k8s

As I understand, technically they don't have to be the same. Maybe let's use a more timestamps in core dagster tests? They don't have to be k8s timestamps here.

Reply:
Done. The `_process_log_stream` is still duplicated however.